hazelcast/proxy/vector_collection.py

```diff
@@ -142,7 +159,8 @@ def _search_near_vector_internal(
         *,
         include_value: bool = False,
         include_vectors: bool = False,
-        limit: int = -1
+        limit: int,
+        hints: Dict[str, str] = None
```

Comment:
I think we shouldn't add hints to the Python API.
But let's discuss.

Reply:
as discussed today: hints will be available in python API, `partitionLimit` hint vs explicit parameter to discuss (outside this PR), better validation/conversion of hints (eg. converting number values to strings) can be implemented also outside this PR